calcom/synclinear.com
# Make Linear deployment button more flexible

# Summary

- Allow Linear deployment button to recover a deleted webhook
- Check for existing webhook rather than only existence in DB

## Test Plan

- Create a sync. Under Linear > Settings > API > Webhooks, delete the webhook. Refresh the UI. Deploy button should show as not deployed.
- Re-deploy. Webhook should be recovered without affecting DB.
- Repeat but deleting the team from the DB. The team should be re-added to DB without affecting the webhook.

## Related Issues

Closes #96
